client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -1,12 +1,163 @@
+import { Button } from '@automattic/components';
+import { useCallback, useState } from 'react';
+import Modal from 'react-modal';
 import { SelectorProduct } from '../types';
+import { Icons } from './icons/product-icons';
+import { Tags } from './icons/tags';
 import './style.scss';
 
 type Props = {
 	product: SelectorProduct;
+	isVisible: boolean;
+	onClose: () => void;
 };
 
-const ProductLightbox: React.FC< Props > = ( { product } ) => {
-	return <div className="product-lightbox__main-wrapper">{ product.displayName }</div>;
+type Option = {
+	value: string;
+	label: string;
+	checked?: boolean;
+};
+type OptionProps = {
+	options: Option[];
+	onChange: ( value: string ) => void;
+};
+
+const Options: React.FC< OptionProps > = ( { options, onChange } ) => {
+	return (
+		<>
+			{ options.map( ( option ) => {
+				return (
+					<label key={ option.value } className="product-lightbox__variants-option-label">
+						<input
+							type="radio"
+							name="variant-select[]"
+							value={ option.value }
+							checked={ option.checked }
+							onChange={ ( el ) => {
+								onChange( el.target.value );
+							} }
+						/>
+						<span>{ option.label }</span>
+					</label>
+				);
+			} ) }
+		</>
+	);
+};
+
+const ProductLightbox: React.FC< Props > = ( { product, isVisible, onClose } ) => {
+	const close = useCallback( () => onClose?.(), [ onClose ] );
+
+	const [ checked, setChecked ] = useState( '1' );
+
+	return (
+		<Modal
+			className="product-lightbox__modal"
+			overlayClassName="product-lightbox__modal-overlay"
+			isOpen={ isVisible }
+			onRequestClose={ close }
+			htmlOpenClassName="ReactModal__Html--open"
```

Comment:
Does this need to be uppercase?

Reply:
Hmm, I'm not sure. I actually copied this one from the 'Dialog' component so it would have similar behavior